libraries/standard/mqtt/include/mqtt.h

```diff
@@ -406,6 +406,26 @@ MQTTStatus_t MQTT_ReceiveLoop( MQTTContext_t * pContext,
  */
 uint16_t MQTT_GetPacketId( MQTTContext_t * pContext );
 
+/**
+ * @brief Parses the payload of an MQTT SUBACK packet that contains the status
+ * code of subscription request made for topic filters in the equivalent
+ * subscribe packet.
+ *
+ * This function can be used to iterate over the status code
```

Comment:
Can it? It doesn't seem so

Reply:
Oops, this is an artifact of an alternative approach idea I had in mind of the API. Will remove it.